sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-b2c-oidc/src/main/java/sample/aad/security/WebSecurityConfiguration.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
```

Comment:
Our license headers are different - please follow the following style:

```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Reply:
Changed.